utils/utils.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+import "github.com/flanksource/commons/logger"
+
+func LogIfError(err error, description string) {
```

Comment:
Can we move this to a log package so that it becomes `log.IfError` and `log.IfErrorf`